ocs_samples/basic_samples/SDS/JavaScript/Angular/readme.md

```diff
@@ -203,22 +225,92 @@ insertValues(streamId: string, events: Array<any>) {
     }
 ```
 
+First the event is created locally by populating a newWave event as follows:
```

Comment:
should this say "new WaveData event or are you referring to newWave as a sort of variable name?

Reply:
Yes you're right, its a new (variable) waveData event in Angular, in Python and Node its a (variable) newWave event.